DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/userpreferences/userpreferences.controller.js

```diff
@@ -9,15 +9,26 @@
 
     function UserPreferencesController(userData, logger, API_URL) {
         var vm = this;
+        vm.imageSource = ''; 
 
         activate();
 
         function activate() {
+            uploadPhoto();
             logger.info('Activate User Preferences View');
         }
 
+        function uploadPhoto() {
+            return userData.getImage(API_URL.UPLOADPHOTO).then(function (newImageUrl) {
```

Comment:
I have the same url for post and get request. And both requests return image url.